### PR TITLE
Adding List Content

### PR DESCRIPTION
**Pull request by Woolfe-Glitch**
Just Moving over the List writing, will be adding images shortly.

### CODE REVIEW

**Comment by garrettdigital** [comment]
Selected Item section is more of a "state" than a type of list. I suggest moving this content down to the Selected/Active state section.

**Comment by garrettdigital** [comment]
The various states could benefit from having screen shots included to demonstrate what they look like.

File: docs/design-language.md [hunk 1]
```diff
@@ -114,19 +108,25 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo eli
...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo elit. Curabitur dui nulla, vulputate id rutrum sed, maximus consequat eros. Nullam pharetra urna quis sapien tincidunt, et iaculis mauris ornare. Vestibulum convallis et felis at lobortis. Nulla bibendum vel quam quis accumsan.
+A few guidelines should be followed to ensure that the list element looks uniform, and fit within the application properly.
 
 ### Hover
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo elit. Curabitur dui nulla, vulputate id rutrum sed, maximus consequat eros. Nullam pharetra urna quis sapien tincidunt, et iaculis mauris ornare. Vestibulum convallis et felis at lobortis. Nulla bibendum vel quam quis accumsan.
+![list](/img/list/selected-item.png "Selected Item List")
+
+When a hover state is evoked, it cannot rely on simply a colour change. There must be a stage change that shows there is a differentiation to pull them into WCAG compliance. There is typically a background colour shift, as well as a side border colour added. Hover states will always include a styling change, and the background colour will get a gradient.
```
Comment (garrettdigital) [inline docs/design-language.md:117]: "evoked" should be "invoked".

Comment (garrettdigital) [reply docs/design-language.md:117]: "stage change" should be "state change"

File: docs/design-language.md [hunk 2]
```diff
@@ -105,28 +99,32 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo eli
...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo elit. Curabitur dui nulla, vulputate id rutrum sed, maximus consequat eros. Nullam pharetra urna quis sapien tincidunt, et iaculis mauris ornare. Vestibulum convallis et felis at lobortis. Nulla bibendum vel quam quis accumsan.
+A few guidelines should be followed to ensure that the list element looks uniform, and fit within the application properly.
 
 ### Hover
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo elit. Curabitur dui nulla, vulputate id rutrum sed, maximus consequat eros. Nullam pharetra urna quis sapien tincidunt, et iaculis mauris ornare. Vestibulum convallis et felis at lobortis. Nulla bibendum vel quam quis accumsan.
+![list](/img/list/selected-item.png "Hover Item State")
+
+When a hover state is triggered, it cannot just rely on a color change. There also must be a state change that shows the differentiation. For example, in our programs, there is a background color change, as well as the addition of a side border color. Hover states always include a styling change, and the background color will get a gradient.
```
Comment (garrettdigital) [inline docs/design-language.md:115]: Can I suggest adding in some wording to explain why we can't just use color when hovering? Something along the lines of, "A hover state cannot rely on a color change as the only indication that it is being hovered due to users that may be colorblind or have other visual impairments. In our applications we will typically add a border along with a color change to be WCAG compliant."

**Review by garrettdigital — CHANGES_REQUESTED** [review]

**Review by garrettdigital — APPROVED** [review]

**Review by jfgodin — APPROVED** [review]

**Review by garrettdigital — CHANGES_REQUESTED** [review]
"States" are showing up as a top level item in the right side menu. I assume it's just an issue with the heading levels that needs to be corrected.

**Review by garrettdigital — APPROVED** [review]

**Pull request merged**